runtime/lua/vim/treesitter/_meta.lua

```diff
@@ -16,8 +16,8 @@
 ---@field named_child_count fun(self: TSNode): integer
 ---@field child fun(self: TSNode, integer): TSNode?
 ---@field named_child fun(self: TSNode, integer): TSNode?
----@field descendant_for_range fun(self: TSNode, integer, integer, integer, integer): TSNode?
----@field named_descendant_for_range fun(self: TSNode, integer, integer, integer, integer): TSNode?
+---@field descendant_for_range fun(self: TSNode, start_lnum: integer, start_col: integer, end_lnum: integer, end_col: integer): TSNode?
```

Comment:
`child` and `named_child` are broken too. Can we also fix those?

Reply:
Have pushed the fix.